thesis-de/content-de.tex

```diff
@@ -2942,6 +2943,36 @@ \section{Unit-Tests}
 \end{outline}
 \mr%
 
+\section{Akzeptanz Test}
+Ein Akzeptanz Test prüft den zuvor erstellten Code in einer eigens dafür
+gestarteten virtuellen Maschine oder abgesichertem Container mit einem
+Betriebssystem auf die Funktionalität der einzelnen Module. Hierzu wurden schon
+bereits vorhandene Test Szenarien von \gls{testkitchen} und \gls{serverspec}
+verwendet und diese auf die im Projekt verwendeten Module angepasst. Der Code
+wird innerhalb dieser Tests ausgeführt und geprüft, ob die Grundfunktion, sowie
+die Funktionalität der einzelnen Module gegeben sind. Diese Szenarien prüfen
+beispielsweise, ob die Erhaltung der Daten, sowie speichern dieser Daten in der
+Datenbank und Visualisierung der Graphen funktioniert. Die Ausgabe erfolgt hier
+über einen booleschen Wert, welcher entweder \texttt{true} oder \texttt{false}
+sein kann. Anschließend kann der Entwickler diesen Code erneut anpassen und
+testen. Nachfolgend sind zwei Tests für einen Akzeptanztest für das Modul
+Collectd angehangen. Der vollständige Test ist dem
+Anhang~\ref{lst:collectd-test}beigefügt.
+
```

Comment:
in dem abschnitt sollte noch erwähnt werden, das nicht nur die einzelnen dienste auf dem system geprüft werden, sondern auch ob puppet idempotent ist und fehlerfrei läuft.